capstone/src/main/java/com/google/step/tests/LoginServletTest.java

```diff
@@ -0,0 +1,186 @@
+package com.google.step.tests;
```

Comment:
Typically, Java tests are put in a directory called `/test` on the same level as main. So you'd have a `src/main/java/...` and a `src/test/java/...`. And the package for the tests would be the exact same as they are in the main files. So in this case, the test would be in package `com.google.step.servlets`. This makes it easy to find corresponding test files without having cluttered folders or a complicated folder structure in smaller projects.